app/bundles/CampaignBundle/Tests/Executioner/InactiveExecutionerTest.php

```diff
@@ -5,44 +5,52 @@
 use Doctrine\Common\Collections\ArrayCollection;
 use Mautic\CampaignBundle\Entity\Campaign;
 use Mautic\CampaignBundle\Entity\Event;
+use Mautic\CampaignBundle\Entity\LeadRepository;
 use Mautic\CampaignBundle\Executioner\ContactFinder\InactiveContactFinder;
 use Mautic\CampaignBundle\Executioner\ContactFinder\Limiter\ContactLimiter;
 use Mautic\CampaignBundle\Executioner\EventExecutioner;
+use Mautic\CampaignBundle\Executioner\Helper\EventRedirectionHelper;
 use Mautic\CampaignBundle\Executioner\Helper\InactiveHelper;
 use Mautic\CampaignBundle\Executioner\InactiveExecutioner;
 use Mautic\CampaignBundle\Executioner\Scheduler\EventScheduler;
 use Mautic\CoreBundle\ProcessSignal\ProcessSignalService;
 use Mautic\CoreBundle\Translation\Translator;
 use Mautic\LeadBundle\Entity\Lead;
+use PHPUnit\Framework\MockObject\MockObject;
 use Psr\Log\NullLogger;
 use Symfony\Component\Console\Output\BufferedOutput;
 
 class InactiveExecutionerTest extends \PHPUnit\Framework\TestCase
 {
     /**
-     * @var \PHPUnit\Framework\MockObject\MockObject|InactiveContactFinder
+     * @var MockObject|InactiveContactFinder
      */
-    private \PHPUnit\Framework\MockObject\MockObject $inactiveContactFinder;
+    private MockObject $inactiveContactFinder;
 
     /**
-     * @var \PHPUnit\Framework\MockObject\MockObject|Translator
+     * @var MockObject|Translator
      */
-    private \PHPUnit\Framework\MockObject\MockObject $translator;
+    private MockObject $translator;
 
     /**
-     * @var \PHPUnit\Framework\MockObject\MockObject|EventScheduler
+     * @var MockObject|EventScheduler
      */
-    private \PHPUnit\Framework\MockObject\MockObject $eventScheduler;
+    private MockObject $eventScheduler;
 
     /**
-     * @var \PHPUnit\Framework\MockObject\MockObject|InactiveHelper
+     * @var MockObject|InactiveHelper
      */
-    private \PHPUnit\Framework\MockObject\MockObject $inactiveHelper;
+    private MockObject $inactiveHelper;
 
     /**
-     * @var \PHPUnit\Framework\MockObject\MockObject|EventExecutioner
+     * @var MockObject|EventExecutioner
      */
-    private \PHPUnit\Framework\MockObject\MockObject $eventExecutioner;
+    private MockObject $eventExecutioner;
+
+    /**
+     * @var MockObject&EventRedirectionHelper
+     */
+    private $redirectionHelper;
```

Comment:
You can even do this in M7:
```suggestion
    private MockObject&EventRedirectionHelper $redirectionHelper;
```

Reply:
Fixed.